static/app/components/events/eventTagsAndScreenshot/screenshot.tsx

```diff
@@ -0,0 +1,164 @@
+import {Fragment, useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {Client} from 'app/api';
+import MenuItemActionLink from 'app/components/actions/menuItemActionLink';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import DropdownLink from 'app/components/dropdownLink';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import {Panel, PanelBody, PanelFooter} from 'app/components/panels';
+import {IconDownload, IconEllipsis} from 'app/icons';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {EventAttachment, Organization, Project} from 'app/types';
+import {Event} from 'app/types/event';
+import withApi from 'app/utils/withApi';
+
+import DataSection from './dataSection';
+import ScreenshotEmpty from './screenshotEmpty';
+import ScreenshotPreview from './screenshotPreview';
+
+type Props = {
+  event: Event;
+  api: Client;
+  orgSlug: Organization['slug'];
+  projectSlug: Project['slug'];
+};
+
+function Screenshot({event, api, orgSlug, projectSlug}: Props) {
+  const [attachments, setAttachments] = useState<EventAttachment[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  async function fetchData() {
+    if (!event) {
+      return;
+    }
+
+    setIsLoading(true);
+
+    try {
+      const response = await api.requestPromise(
+        `/projects/${orgSlug}/${projectSlug}/events/${event.id}/attachments/`
+      );
+      setAttachments(response);
+      setIsLoading(false);
+    } catch (_err) {
+      // TODO: Error-handling
+      setAttachments([]);
+      setIsLoading(false);
+    }
+  }
+
+  function hasPreview(attachment: EventAttachment) {
+    switch (attachment.mimetype) {
+      case 'text/plain':
+        return attachment.size > 0;
+      case 'text/json':
+      case 'text/x-json':
+      case 'application/json':
+        return true;
+      case 'image/jpeg':
+      case 'image/png':
+      case 'image/gif':
+        return true;
+      default:
+        return false;
+    }
+  }
```

Comment:
This screenshot preview should work only for images. All other attachments should still be visible/previewable down in the `EventAttachments` section.

Reply:
Robins design has a video player, so I thought we wanted to support everything. I'll update the code